src/NSwag.MSBuild/NSwag.MSBuild.nuspec

```diff
@@ -34,5 +34,6 @@
     <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp2.2\Publish\**" target="tools/NetCore22" />
     <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp3.0\Publish\**" target="tools/NetCore30" />
     <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp3.1\Publish\**" target="tools/NetCore31" />
+    <file src="..\NSwag.ConsoleCore\bin\release\net5.0\Publish\**" target="tools/NetCore31" />
```

Comment:
It should be NetCore50.

Reply:
Then we would need to change the default behavior?... net5.0 is the new name - netcoreapp has been renamed by Microsoft?

Are you sure we should do that?

Reply:
I guess he means target should be 

``` CSharp
target="tools/Net50"
```

Reply:
You are right - thanks for the hint...

Fixed that ;-)